.gitignore

```diff
@@ -1,33 +1,2 @@
-# Logs
-logs
-*.log
-npm-debug.log*
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# node-waf configuration
-.lock-wscript
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
-# Dependency directory
 node_modules
-
-# Optional npm cache directory
-.npm
-
-# Optional REPL history
-.node_repl_history
```

Comment:
The trouble with .gitignore is that it often becomes quite specific to the individual project. There are no limits on what tools a developer can use. Therefore I decided to keep it sparse. However, I think you are right about the istanbul coverage, as it is something we are using too, and would be a useful addition to all the repos.